README.md

```diff
@@ -72,23 +71,34 @@ docker_image_manifest_path = ""   # Path to manifest in docker image
 The default configuration of _dbtwiz_ will be installed the first time you run it, but you
 may want to adjust some settings from the get-go to fit your environment.
 
-#### Dark mode
-If you're using a dark background colour in your terminal, you should configure _dbtwiz_ to
-use bright colours for highlighting in previews and elsewhere to make the text more readable.
-
-Run the following command to switch from default light mode to dark mode:
-```shell
-$ dbtwiz config theme dark
-```
-
-#### Preview formatter
-
-By default, _dbtwiz_ uses the command _fmt_ tool to format text in the preview window when
-selecting models interactively. Under macOS, the _fmt_ tool won't handle ANSI escape codes,
-and unless you have the GNU coreutils version of _fmt_ you will get garbage characters in the
-preview window, and should switch to the simple _cat_ command for formatting instead:
-```shell
-$ dbtwiz config model_info:formatter "cat -s"
+The config settings are stored in a file `config.toml` in the `dbtwiz` folder within
+your user's app settings directory. In a GitHub Codespace or in a local Linux environment
+the file is located at `~/.config/dbtwiz/config.toml`, while in a local Windows environment
+it's located at `%appdata%/dbtwiz/config.toml`. TODO: Where is it located in MacOS?
+
+If dbtwiz looks up a user config file and it doesn't exist, it will be created with
+the following default settings:
+
+```toml
+# When true, check for existing GCP auth token, and ask for
+# automatic reauthentication if needed.
+auth_check = true
```

Comment:
Any reason why this shouldn't always happen?

Reply:
Are we sure the `auth` module handles every scenario, ie. `APPDATA` in env otherwise `~/.config`? Need to check this on MacOS, I guess.